docs/assets/scripts/docsearch.sbt.js

```diff
@@ -29,7 +29,7 @@ try {
     container: "#search-algolia",
     appId: "9H1PGK6NP7",
     indexName: "determined-" + version,
-    apiKey: "18b6f7b0b2e20a6bdb00b660ff45d3b8",
+    apiKey: "c2efc1c44ce448fa3425bbbd8cf17d57",
```

Comment:
the old api key was my personal account's key, the new one belongs to our CI account.

But... this isn't just for CI, it's actually for our docs frontend.  Maybe we should make a new algolia account (with a new email) to make it clearer that this key should never get rotated.